tests/transports-integrations/tests/utils/config_loader.py

```
@@ -0,0 +1,309 @@
+        """Get API configuration (timeout, retries, etc.)"""
+        return self._config["api"]
+
+    def get_test_settings(self) -> Dict[str, Any]:
+        """Get test configuration settings"""
+        return self._config["test_settings"]
+
+    def get_integration_settings(self, integration: str) -> Dict[str, Any]:
+        """Get integration-specific settings"""
+        return self._config["integration_settings"].get(integration, {})
+
+    def get_environment_config(self, environment: str = None) -> Dict[str, Any]:
```

Ruff (0.11.9)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)